tox.ini

```diff
@@ -25,36 +25,6 @@ passenv =
 commands =
     py.test --cov=scrapy --cov-report= {posargs:scrapy tests}
 
-[testenv:trusty]
-basepython = python2.7
-deps =
-    pyOpenSSL==0.13
-    lxml==3.3.3
-    Twisted==13.2.0
-    boto==2.20.1
-    Pillow==2.3.0
-    cssselect==0.9.1
-    zope.interface==4.0.5
-    -rtests/requirements-py2.txt
-
-[testenv:jessie]
-# https://packages.debian.org/en/jessie/python/
-# https://packages.debian.org/en/jessie/zope/
-basepython = python2.7
-deps =
-    cryptography==0.6.1
-    pyOpenSSL==0.14
-    lxml==3.4.0
-    Twisted==14.0.2
```

Comment:
Without version pinning we'll be only testing latest Twisted, etc. version. Is it intended? If yes, probably setup.py should be updated, as we're allowing early Twisted versions there, which we no longer test for. If no, it could make sense to have an environment with pinned package versions, the ones which we support - it doesn't have to be tied to Ubuntu or Debian distributions, but it can look very similar.